src/object.cpp

```diff
@@ -1605,6 +1606,10 @@ int Object::ScoreDefSetCurrent(FunctorParams *functorParams)
         if (staff->IsTablature()) {
             staff->m_drawingStaffSize *= TABLATURE_STAFF_RATIO;
         }
+        if (MeterSigGrp *metersiggrp = params->m_currentStaffDef->GetCurrentMeterSigGrp();
+            metersiggrp->GetFunc() == meterSigGrpLog_FUNC_alternating) {
+            metersiggrp->AddMeasureIdToVector(staff->GetParent()->GetUuid());
```

Comment:
You should do `staff->GetFirstAncestor(MEASURE)` to make sure it is a measure and not editorial markup